.flake8

```diff
@@ -0,0 +1,8 @@
+[flake8]
+ignore = F403
+max-line-length = 88
+max-complexity = 10
+exclude = 
```

Comment:
Just a suggestion. Instead of adding exclusions, can we move all the python source files to a subdirectory (say `sunnah_api/` for example)? Our root directory is now pretty crowded so this might be a good opportunity to do this.
We can maybe do this in a later PR too if it's too complicated.

Reply:
I think its better to have 2 separate PRs, i make another PR for that inshallah!